test/fixtures/dialects/clickhouse/cte_columns.sql

```diff
@@ -0,0 +1,2 @@
+WITH t(col1, col2) AS (SELECT 1, 2 FROM foo)
+SELECT * from t;
```

Comment:
Nit, newline - that minus symbol GitHub uses for this always bugs me for some reason!! Probably why they use it 😁
<img width="143" alt="image" src="https://user-images.githubusercontent.com/10931297/178343838-2996a9d4-9dae-4f6b-b62b-122454151f7f.png">

```suggestion
SELECT * from t;

```